client/app/cda/details/home/detail/edit/cda-detail-edit.html

```diff
@@ -1,63 +1,38 @@
-<form id="editDetailForm" name="editDetailForm" class="oui-modal oui-modal_shadow">
-    <div class="oui-modal__header">
-        <button class="oui-icon oui-icon-close_thin oui-modal__close-button" type="button" aria-label="Exit"
-                data-ng-click="CdaDetailEditCtrl.closeModal()"></button>
-    </div>
-    <div class="oui-modal__body oui-modal__body_no-icon">
-        <div class="oui-modal__body-text" data-ng-if="CdaDetailEditCtrl.saving">
-            <div class="text-center">
-                <oui-spinner></oui-spinner>
-            </div>
+<form id="editDetailForm" name="editDetailForm">
+    <oui-modal
+        data-title="{{ ::'cda_detail_edit_title' | translate }}"
+        data-primary-action="editDetailForm.$valid && CdaDetailEditCtrl.editCluster()"
```

Comment:
Primary button should be a submit type and the action should be applied on the form with ng-submit.
A discussion is required